src/ProjectService.Data/UserRepository.cs

```diff
@@ -106,5 +100,31 @@ public void Remove(Guid userId)
 
             _provider.Save();
         }
+
+        public bool AreExist(params Guid[] ids)
+        {
+            return AreExist(null, null, ids);
+        }
+
+        public bool AreExist(bool? isManager, Guid? projectId, params Guid[] ids)
+        {
+            if (isManager == null)
+            {
+                var dbIds = _provider.ProjectsUsers.Select(x => x.UserId);
+                return ids.All(x => dbIds.Contains(x));
+            }
+            else
+            {
+                DbProjectUser users = _provider.ProjectsUsers.Where(user => user.UserId == ids.First() && user.ProjectId == projectId).First();
+                if ((bool)isManager)
```

Comment:
Why is isManager nullable variable? And use HasValue and  Value

Reply:
rewrite